base/reflection.jl

```diff
@@ -330,7 +330,9 @@ end
 """
     objectid(x) -> UInt
 
-Get a hash value for `x` based on object identity. `objectid(x)==objectid(y)` if `x === y`.
+Get a hash value for `x` based on object identity.
+
+If `x === y` then `objectid(x) == objectid(y)`, and ususally when `x !== y`, `objectid(x) != objectid(y)`.
```

Comment:
```suggestion
If `x === y` then `objectid(x) == objectid(y)`, and usually when `x !== y`, `objectid(x) != objectid(y)`.
```

Also just from curiosity, when do we want `objectid(x) != objectid(y) && x !== y`?